### PR TITLE
feat: my quick links

### PR DESCRIPTION
**Pull request by setchy**
simple update to add the following "quick links" to the Sidebar: 

- my issues
- my pull requests

 
Screenshot 2024-06-17 at 3 52 58 PM

Screenshot 2024-06-17 at 3 41 34 PM

Closes https://github.com/gitify-app/gitify/issues/498 (at least in a way that's perhaps sufficient for now)

Feedback welcomed on Sidebar layout. I did try side-by-side but it felt awkward

### CODE REVIEW

**Comment by setchy** [comment]
In time we may consider pulling the datasets and displaying within Gitify 🤔

**Review by bmulholland — APPROVED** [review]

File: src/components/Sidebar.tsx [hunk 1]
```diff
@@ -59,7 +71,25 @@ export const Sidebar: FC = () => {
             size={12}
             aria-label={`${notificationsCount} Unread Notifications`}
           />
-          {notificationsCount > 0 && notificationsCount}
+          {hasNotifications && notificationsCount}
+        </button>
+
+        <button
+          type="button"
+          className="my-1 flex cursor-pointer items-center justify-around self-stretch px-2 py-1 text-xs font-extrabold text-white"
```
[the remark this answers is not in the document]
Comment (setchy) [reply src/components/Sidebar.tsx:79]: Good call - I've extracted it into a new `SidebarButton` component

[the remark this answers is not in the document]
Comment (setchy) [reply src/components/Sidebar.tsx]: https://github.com/gitify-app/gitify/pull/1270

File: src/components/Sidebar.tsx [hunk 2]
```diff
@@ -56,6 +63,18 @@ export const Sidebar: FC = () => {
           icon={BellIcon}
           onClick={() => openGitHubNotifications()}
         />
+
+        <SidebarButton
```
Comment (bmulholland) [inline src/components/Sidebar.tsx:67]: This is easy now!

**Review by bmulholland — APPROVED** [review]

File: src/components/Sidebar.tsx (hunk 2 above)
Comment (setchy) [reply src/components/Sidebar.tsx:67]: _too_ easy 😜

**Review by afonsojramos — APPROVED** [review]

**Comment by afonsojramos** [comment]
To be honest, I was initially a bit hesitant on adding more icons/buttons, but these are small I suppose. Looking good!

**Comment by bmulholland** [comment]
Yeah, on the one hand it's more visual stuff, on the other hand, arguably useful? Wish we could get stats on whether these are used. I think any form of user analytics would get hated on from an OSS project, sadly.

**Pull request merged**